diem-move/diem-framework/experimental/Move.toml

```diff
@@ -5,7 +5,7 @@ version = "1.5.0"
 [addresses]
 Std = "0x1"
 DiemFramework = "0x1"
-BARS = "0x34CD440B72D03907100007D7BC0080A1"
+BARS = "0x06505CCD81E562B524D8F656ABD92A15"
```

Comment:
Had to change this so it matches the key generated deterministically using a known seed.